update.py

```diff
@@ -63,6 +63,7 @@ def check_notices(session, sessionData):
         content_div = bs.find_all(content, 'div', {'id': 'printableArea'})[0]
         notice['text'] = content_div.decode_contents(formatter='html')
         notice['time'] = cds[6].string
+        notice['id'] = id_ + "_" + year
```

Comment:
Not an issue but can we call this key `uid` since `id_` refers to the notice's ID

Reply:
yeah okay I'll change the name